### PR TITLE
AP_BoardConfig: Deconflict Aero FC and PX4 BRD_TYPE enums

### PR DESCRIPTION
**Pull request by WickedShell**
Aero FC stole a VRBrain BRD_TYPE value.

### CODE REVIEW

**Comment by magicrub** [comment]
@lucasdemarchi did you see this PR?

File: libraries/AP_BoardConfig/AP_BoardConfig.h [hunk 1]
```diff
@@ -66,7 +66,7 @@ class AP_BoardConfig {
         PX4_BOARD_PIXRACER = 4,
         PX4_BOARD_PHMINI   = 5,
         PX4_BOARD_PH2SLIM  = 6,
-        PX4_BOARD_AEROFC   = 7,
+        PX4_BOARD_AEROFC   = 13,
```
Comment (lucasdemarchi) [inline libraries/AP_BoardConfig/AP_BoardConfig.h]: I don't see any other one with 7 here. Where's the conflict?

Comment (WickedShell) [inline libraries/AP_BoardConfig/AP_BoardConfig.h]: VRBrain uses 7, VRX_BOARD_BRAIN51

Comment (WickedShell) [inline libraries/AP_BoardConfig/AP_BoardConfig.h]: Also the .cpp file documents it as being 13 for aero :)

**Comment by lucasdemarchi** [comment]
I went with a similar patch, but with added comment and commit message.

**Pull request closed without merging**